cypress.config.js

```diff
@@ -9,6 +9,6 @@ export default defineConfig({
     setupNodeEvents(on, config) {},
     specPattern: 'tests/e2e/specs/**/*.cy.{js,jsx,ts,tsx}',
     supportFile: 'tests/e2e/support/index.js',
-    baseUrl: 'http://localhost:3000/',
+    baseUrl: 'http://localhost:5174/',
```

Comment:
Vite 3 has changed the default port, so as not to clash with other frameworks (such as Rails)